WordPressKit/DomainsServiceRemote.swift

```diff
@@ -26,6 +26,27 @@ public class DomainsServiceRemote: ServiceRemoteWordPressComREST {
         case decodingFailed
     }
 
+    public enum DomainSuggestionType {
+        case noWordpressDotCom
+        case includeWordPressDotCom
+        case onlyWordPressDotCom
+        
+        fileprivate func parameters() -> [String: AnyObject] {
+            var result: [String: AnyObject] = [:]
```

Comment:
nitpick: if you don't assign an initial value, this could be a `let`

Reply:
Done